script.py

```diff
@@ -0,0 +1,239 @@
+"""To use this script, install the provided requirements.txt file and
+supply a secrets.py file with the required parameters listed below.
+
+JSON records will be written into the specified file and HTML reports
+will be written into an outputs filter under the provided directory.
+"""
+
+
+from datetime import datetime, timedelta
+from jinja2 import Template
+from ua_parser import user_agent_parser
+
+import collections
+import json
+import os
+import requests
+import secrets
+import smtplib
+
+
+# This is the absolute path to the directory where you'd like
+# the JSON and HTML files to be written.
+# The template HTML file should also be placed in this
+# directory.
+DIR = secrets.DIR
+# This is the name of the JSON file that you'd like to write to
+FILENAME = secrets.FILENAME
+# This is the username for your LA1 login
+USERNAME = secrets.USERNAME
+# This is the password for your LA1 login
+PASSWORD = secrets.PASSWORD
+# Email address from which you'd like to send viewer information
+FROM_EMAIL = secrets.FROM_EMAIL
+# Email address to which you'd like to send viewer information
+TO_EMAIL = secrets.TO_EMAIL
```

Comment:
Oooh actually how about if these aren't supplied, we just skip emailing? That would enable use of this script for just the download/storage feature.

Reply:
Oh yeah that's a good idea